html/changelogs/damian.yml

```diff
@@ -0,0 +1,4 @@
+author: DamianQ
```

Comment:
👌

Reply:
Technically would have to be DamianX here.